src/components/EditForm/EditForm.jsx

```diff
@@ -0,0 +1,215 @@
+import React, {
```

Comment:
what is the difference between EditForm and AddForm?
from my point of view it should look same